.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+
+# Thanks for contributing a pull request! 
+## Please make sure you see our contribution guidelines: https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md
+
+## What type of PR is this? (check all applicable)
+
+- [ ] 💾 Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 🔧 Optimization
+- [ ] 📝 Documentation
+- [ ] ✅ Test
+- [ ] 🐳 Other
+
+## Related Issues 
+
+- Related Issue # 
+- Closes #
+
+## About tests
+
+- [ ] Tests added. If you are adding a new feature, please also add a test for it. 
+- [ ] No, I did not add a test.
+
+## If you have comments or can explain your changes, please do so below.
+
```

Comment:
Should we add users or group to ping/add for review? It can be done/suggested here, or achieved via [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

Reply:
@FBruzzesi I have a slight preference for skipping this step to encourage everyone to contribute to code review.

Reply:
for now activity is low enough that it's easy enough to follow everything - maybe we can add this if/when activity becomes too high for any one person to follow?